src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -260,6 +266,43 @@ public sealed partial class JsonNumber : System.Text.Json.JsonNode, System.IEqua
         [System.CLSCompliantAttribute(false)]
         public bool TryGetUInt64(out ulong value) { throw null; }
     }
+    public sealed partial class JsonObject : System.Text.Json.JsonNode, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, System.Text.Json.JsonNode>>, System.Collections.IEnumerable
```

Comment:
How can I create a JsonObject from an existing JSON payload (say from a string, or stream)?

Reply:
This is planned to be a method of `JsonNode`: https://github.com/kasiabulat/corefx/blob/master/src/System.Text.Json/src/System/Text/Json/Node/JsonNode.cs

Reply:
(I will work on implementing transformation functions later on)